src/components/layout.tsx

```diff
@@ -120,35 +120,64 @@ const LayoutScaffolding = ({
   );
 };
 
-const Layout = ({ metadata, children, isLandingPage }: Props) => (
-  <StaticQuery
-    query={graphql`
-      query {
-        contentfulHomePage {
-          metadata {
-            title
-            description
-            keywords {
-              keywords
-            }
-            shareImage {
-              file {
-                url
+type MetadataNodes = {
+  node_locale: string;
+  metadata: Props["metadata"];
+};
+
+function getDefaultContentForLocale(
+  locale: string,
+  nodes: MetadataNodes[]
+): Props["defaultContent"] {
+  for (let node of nodes) {
+    if (node.node_locale.startsWith(locale)) {
+      return { metadata: node.metadata };
+    }
+  }
+  throw new Error(
+    `Unable to find default content metadata for locale "${locale}"`
+  );
+}
```

Comment:
Nice! This helper function is great. I could imagine a scenario where we may actually want to abstract this out further, and have this be a global helper function that returns a localized version of content from an array of Contentful nodes— especially if we start to have more non-page components that query Contentful and need to be internationalized. Until then, I think it's fine to keep it as is here, since this is the only place we use the function.